package.json

```diff
@@ -28,6 +28,7 @@
     "@sheerun/mutationobserver-shim": "^0.3.3",
     "@testing-library/jest-dom": "^5.3.0",
     "@testing-library/react": "^10.0.2",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
Added for testing the select dropdown

https://testing-library.com/docs/ecosystem-user-event/#selectoptionselement-values-options